src/pages/contact.js

```diff
@@ -1,18 +1,66 @@
+        setExpanded(expanded === index ? null : index);
+    };
+
     return (
-        <main>
-            <h1>CS (Data) Chapter</h1>
-            <p>
-                isc@datasektionen.se
-            </p>
-            <h1>Media Chapter</h1>
-            <p>
-                international@medieteknik.com
-            </p>
+        <main className="contact-page body">
+            <section className="chapter-contacts col-2">
+                <div className="contact-info">
+                    <h1>CS (Data) Chapter</h1>
+                    <p>isc@datasektionen.se</p>
+                </div>
+                <div className="contact-info">
+                    <h1>Media Chapter</h1>
+                    <p>international@medieteknik.com</p>
+                </div>
+            </section>
+
+            <section className="in-tem">
+                <h2>International Committee Members</h2>
+                <ul>
+                    <li><span className="name">Håvard Alstadheim</span> <span className="role">- President</span></li>
+                    <li><span className="name">Dmitry Chirin, Håvard Alstadheim</span> <span className="role">- Head of events</span></li>
+                    <li><span className="name">Rafael Oliveira, Elena Pan</span> <span className="role">- Head of communication</span></li>
+                    <li><span className="name">Linn Benali</span> <span className="role">- Secretary</span></li>
+                    <li><span className="name">Hugo Tricot</span> <span className="role">- Head of Teambuilding</span></li>
+                    <li><span className="name">Erik Sevelin, Linus Bälter</span> <span className="role">- Head of Website</span></li>
+                    <li><span className="name">Håvard Alstadheim, Rafael Oliveira, Dmitry Chirin</span> <span className="role">- Head of Reception</span></li>
+                </ul>
+            </section>
+
+
+            <section className="FAQ">
+                <h2>Frequently Asked Questions (FAQ)</h2>
+
+                <div 
+                    className={`FAQ-content ${expanded === 0 ? 'expanded' : ''}`} 
+                    onClick={() => toggleFAQ(0)}
+                    aria-expanded={expanded === 0}
+                >
+                    <h3>Who are you? Who is MAIN? What is a chapter?
+                    </h3>
+                    {expanded === 0 && <p className="FAQ-answer">A chapter is a student union associated with a program at KTH, and for example the CS Chapter is then of course associated with the CS program. This website is run by the International Committee, shared between the two chapters for CS and Media. The main goal of the International Committee is to welcome international students into chapter life in general. Feel very free to contact us for more information. THS MAIN is an association that organizes events and more, similarly to chapters, but only for international and master’s students from all programs together.
```

Comment:
Please note CS stands for Computer Science and Media stands for Media Technology the first time in this paragraph.
IN is also charged to help Swedes to go in exchange-studies outside of Sweden. Please add this.
I think the presentation of THS MAIN is good content-wise, but I'd like a rework of the wording to be clearer. Proposition (to be discussed/criticized): "THS MAIN (standing for MAster's and INternational Group) is an association inside THS that organizes events for international students at the for all of KTH throughout the year, while we focus our events on CS and Media's students."
I'm not sure if we should clear out that they do not do the international reception, it's another group that is in charge of it.

Reply:
Will clarify the wording CS and Media, yes i agree of the explanation of MAIN will fix it according to your proposition!

Reply:
Trailing duplicate dot at the end of the paragraph, good otherwise.